src/data/adventures.txt

```diff
@@ -705,3 +705,23 @@ PirateRealm Island	adventure=531	DiffLevel: mid Env: outdoor Stat: 0 nowander	St
 PirateRealm Island	adventure=531	DiffLevel: mid Env: outdoor Stat: 0 nowander	Red Roger's Fortress
 PirateRealm Island	adventure=531	DiffLevel: mid Env: outdoor Stat: 0 nowander	Jack's Hideout
 PirateRealm Island	adventure=531	DiffLevel: mid Env: outdoor Stat: 0 nowander	The Temple
+
+# Obselete areas that nevertheless are reported as possibilities for the autumnaton
+Plains	adventure=18	DiffLevel: unknown Env: unknown Stat: 10	Degrassi Knoll OBE
```

Comment:
Can these be consistently `(OLD)` (or consistently one thing)? Is OBE obsolete?

Reply:
OBE is overtaken by events which apparently doesn't mean that to people who have not worked around the US military.  I originally did not have OBE in the names and about 10 tests failed.  I guessed the tests expected names to be unique and so added something that would identify "my" additions if not.  I'll make consistent.